File: observer/Samuel Sales/DesafioObserver_Samuel/src/Revistas/Galileu.java
package Revistas;

import agencia.Agencia;

/**
 *
 * @author Samuel Sales - Facul
 */
public class Galileu implements ObserverEditoraAbril {

    @Override
    public void atualizar(Agencia entrada) {
        System.out.println(entrada.getNome() + ": Notícias da Revista Galileu!!");
    }

}

File: observer/Renny/PadraoObserverII/Revistas/Galileu.cs
using PadraoObserverII.Interface;
using System;

namespace PadraoObserverII.Revistas
{
    class Galileu : IObservar
    {
        public void Atualizar(Agencia entrada)
        {
            Console.WriteLine(entrada.Nome + ": Notícias da Revista Galileu!");
        }
    }
}
